### DISCUSSION TITLE
When the inverter is in sleep mode.

### DISCUSSION THREAD

**Discussion by Marchellos** [wlcrs/huawei_solar#296] [Ideas]
When the inverter goes into sleep mode, all sensors are no longer available.
Although the inverter is in sleep mode, I would still like to use certain sensors.

Think of total yield.

Is it a stupid request to make this data (last value) also available when the inverter is in sleep mode?
Or is it better to solve this myself in the Yaml file?

**Comment by wlcrs** [comment]
I've experimented with this but decided against it for various reasons. Use a template sensor instead.

**Reply by Marchellos** [reply]
Thanks for your reply.
I think I understand that decision.
I have already created a sensor template for this.
